package.json

```diff
@@ -22,6 +22,7 @@
     "cac": "^5.0.10",
     "chalk": "^2.4.1",
     "cross-spawn": "^6.0.5",
```

Comment:
Wanna take this out if it isn't being used anymore?

Reply:
oh yea, forgot about that..